### PR TITLE
Migrate Hive servlets to spring

### PR DESCRIPTION
**Pull request by ebyhr**
https://user-images.githubusercontent.com/6237050/102953992-e2679100-4515-11eb-84ad-5ce29537c96e.mov

### CODE REVIEW

File: src/main/java/yanagishima/servlet/HiveServlet.java [hunk 1]
```diff
@@ -3,97 +3,85 @@
...
-
-    @Override
-    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-        Map<String, Object> reponseBody = new HashMap<>();
+    @PostMapping(path = {"hive", "spark"})
+    public Map<String, Object> post(@RequestParam String datasource,
+                                    @RequestParam String engine,
+                                    @RequestParam(name = "user", required = false) Optional<String> hiveUser,
+                                    @RequestParam(name = "password", required = false) Optional<String> hivePassword,
+                                    @RequestParam(defaultValue = "false") boolean store,
```
Comment (wyukawa) [inline src/main/java/yanagishima/servlet/HiveServlet.java:38]: Are there any reasons not to use ```@RequestParam(required = false) String query```  like ```HiveAsyncServlet``` ?

Comment (ebyhr) [reply src/main/java/yanagishima/servlet/HiveServlet.java:39]: I was just missing it. Fixed.

Comment (wyukawa) [reply src/main/java/yanagishima/servlet/HiveServlet.java:39]: `@RequestParam(required = false) String query`?
not `@RequestParam(defaultValue = "false") String query`

Comment (ebyhr) [reply src/main/java/yanagishima/servlet/HiveServlet.java:39]: Sorry, fixed again.

**Review by wyukawa — APPROVED** [review]
LGTM

**Pull request merged**